utils/compare_dependency_constraints.py

```diff
@@ -49,8 +50,25 @@ class Discrepancy(NamedTuple):
     in_native: str | None = None
 
     def __str__(self) -> str:
-        return f"{self.dependency}: {self.in_image or 'missing'} in {self.image}, {self.in_native or 'missing'} in native"
+        return f"{self.dependency}: {self.in_image or 'missing'} in {self.image} image, {self.in_native or 'missing'} in Native image"
 
+def find_library_line_number(lib_name: str, file_path: Path):
+    """
+    Searches for a library in the pyproject.toml or Pipfile file and returns the line number where it is found.
+
+    Parameters:
+    - lib_name: The name of the library to search for.
+    - file_path: The directory containing the pyproject.toml or Pipfile.
+
+    Returns:
+    - The line number containing the library name, or None if the library is not found.
+    """
+    with file_path.open('r') as file:
+        for line_number, line in enumerate(file, start=1):  # Start counting from line 1
```

Comment:
```suggestion
    for line_number, line in enumerate(file_path.read_text().splitlines(), start=1):  # Start counting from line 1
```
unindent the rest too 
(p.s. there may be a more-itertools method that does it, but doesn't really matter)